src/cpg_flow/metamist.py

```diff
@@ -523,13 +524,20 @@ def get_cohort_sgs(cohort_id: str) -> dict:
     #     "name": "CohortName"
     #     "dataset": "DatasetName"
     # }
+
     if len(entries['cohorts']) != 1:
         raise MetamistError('We only support one cohort at a time currently')
 
     if entries.get('data') is None and 'errors' in entries:
         message = entries['errors'][0]['message']
         raise MetamistError(f'Error fetching cohort: {message}')
 
+    cohort_status = entries['cohorts'][0]['status']
+    if cohort_status != 'ACTIVE':
+        raise MetamistError(
+            f'Retrieved cohort is {cohort_status}. Only ACTIVE cohorts are allowed. Please check the input cohort list.'
```

Comment:
Could we also log the cohort id in the error message so they know which cohort is hitting this :)